client/src/unifyfs_api.c

```diff
@@ -122,7 +131,17 @@ unifyfs_rc unifyfs_initialize(const char* mountpoint,
             client->use_local_extents = (bool)b;
         }
     }
-
+    /* Determine if we should track all extents on the node-local clients
```

Comment:
Looks like this code block is a duplicate of the one above.

Reply:
fixed

Reply:
the below block is still a duplicate. the above block should be reverted to update client->use_local_extents

Reply:
now there is no duplication, but the original block that updates `client->use_local_extents` is missing

Reply:
ok added that back.